### ISSUE TITLE
How to unregister push token? Neither unregisterAPNSPushTokenForCurrentUser nor unregisterAPNSPushTokenAllForCurrentUser work for me

### ISSUE THREAD

**Issue by booboothefool** [smilefam/SendBird-SDK-JavaScript#32]
`unregisterAPNSPushTokenForCurrentUser` gives me a 405 when I pass in the current token. 

`unregisterAPNSPushTokenAllForCurrentUser` the callback never fires.

**Comment by sf-craig-park** [comment]
Hi, Thank you for using SendBird for your application!

It seems that you're working with iOS-SDK right?
Could you please share us following information:
- which platform you're developing on, iOS or JS or React-Native
- what SDK version you're working with
- the device token you tried to unregister
(if you mind sharing here, send the device token to craig.park@sendbird.com)
- if you invoked `unregisterAPNSPushTokenForCurrentUser` before disconnect, which means you called the method while connected to SendBird

Thank you!

**Comment by booboothefool** [comment]
I'm using:
```
    "react-native": "0.37.0",
    "sendbird": "^3.0.18",
```

It has been working great so far! However when my user logs out, they still get push notifications because I can't unregister the token. 😛 

* device token: 823a875e79dddb1515b6881d4b70b77589f7e13749b9a7beeb2a22c7d5cb4c5b
* `unregisterAPNSPushTokenForCurrentUser` was called while connected to SendBird

**Comment by booboothefool** [comment]
@sf-craig-park I'm using:
```
    "react-native": "0.37.0",
    "sendbird": "^3.0.18",
```

It has been working great so far! However when my user logs out, they still get push notifications because I can't unregister the token. 😛 

* device token: 823a875e79dddb1515b6881d4b70b77589f7e13749b9a7beeb2a22c7d5cb4c5b
* `unregisterAPNSPushTokenForCurrentUser` was called while connected to SendBird

**Comment by sf-jin-ku** [comment]
It should be fixed in the latest 3.0.22! Could you try it?

Thanks.
